cfgov/v1/feeds.py

```diff
@@ -63,3 +63,29 @@ def serve(self, request, format='html'):
             return FilterableFeed(self, context)(request)
         else:
             return super(FilterableFeedPageMixin, self).serve(request)
+
+
+def get_page_ancestor_with_feed(page):
```

Comment:
I think the naming of this function and its local variables is somewhat confusing, given that the passed `page` itself could be found and returned. Consider renaming or refactoring to eliminate the weird inclusive `get_ancestors` behavior.

Reply:
Thanks for the suggestion @Scotchester. How does the change in c5bfc77 look to you?